### DISCUSSION TITLE
已经在rk3399-pinebook-pro开发板上流畅运行V23的alpha 2 求发布ARM版本时，能有这个正式版镜像。

### DISCUSSION THREAD

**Discussion by hahahawangyu** [linuxdeepin/developer-center#4213] [General | 常规讨论]
已运行近一个月，在不更新的情况下，流畅运行各种应用。
因为移植原镜像为香橙派开发板，默认无电池，所以任务栏插件中，无电池电量图标。
插入手机时，系统变慢并产生卡顿现像。
其他一切正常。

**Comment by Zeno-sole** [comment]
我们暂时不会开发rk3399的镜像，不过相关的卡顿问题在x86平台应该也有，我们会持续迭代解决

**Comment by BLumia** [comment]
可以考虑加入 arm (#deepin-arm-pc:matrix.org) 与移植 (#dde-port:deepin.org) 相关的 SIG，一起讨论相关的移植话题 :)
